Vagrantfile

```
@@ -0,0 +1,23 @@
+# -*- mode: ruby -*-
+# vi: set ft=ruby :
+
+Vagrant.configure("2") do |config|
+  config.vm.box = "hashicorp/bionic64"
+  config.vm.provision "shell", inline: <<-SHELL

```

You'll want to run this script as the vagrant user.

```suggestion
  config.vm.provision "shell", privileged: false, inline: <<-SHELL

```